app/controllers/admin/merchants_controller.rb

```diff
@@ -23,4 +23,12 @@ def update
 
     redirect_to "/admin/merchants/#{merchant.id}?update=true"
   end
+
+  def update_status
+    merchant = Merchant.find(params[:id])
+    merchant.update(status: !merchant.status)
```

Comment:
We might want a hard set here instead of a toggle during refactor in case we get bad data in the table.